### DISCUSSION TITLE
1.8

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#2571] [General]
Release 1.8 is finally here!

View the changelogs [here](https://kadedev.github.io/Kade-Engine/changelogs/changelog-1.8)!

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.8'>1.8</a>.</em>

**Comment by ghost** [comment]
oh shit lets go 1.8 is out

**Reply by asdfd3v** [reply]
BRUH THIS IS SO FUCKING EPIC

**Reply by SusImpostor34** [reply]
epok

**Reply by sosa360** [reply]
EPIC

**Reply by jukeliv** [reply]
but it have 1000000000000000000000000000000000 bugs 😔

**Comment by SanicBTW** [comment]
Holy shit lets gooo

**Comment by lxcianitoc0des** [comment]
FINALLY! Lets go beat Psych Engine!

**Reply by SanicBTW** [reply]
I mean Psych Engine its just a good engine and you can choose between a bunch of engines which practically does the same stuff but every engine adds a lot of new functions

**Reply by remaster2011** [reply]
> I mean Psych Engine its just a good engine and you can choose between a bunch of engines which practically does the same stuff but every engine adds a lot of new functions

why not just make 1 engine that has all features of all engines and is the best engine ever lol

**Reply by fcland** [reply]
> > ฉันหมายถึง Psych Engine มันเป็นเอ็นจิ้นที่ดีและคุณสามารถเลือกระหว่างเอ็นจิ้นที่ทำสิ่งเดียวกันได้จริง แต่ทุกเอ็นจิ้นจะเพิ่มฟังก์ชันใหม่มากมาย
> 
> ทำไมไม่สร้างเอ็นจิ้น 1 อันที่มีคุณสมบัติทั้งหมดของเครื่องยนต์ทั้งหมดและเป็นเอ็นจิ้นที่ดีที่สุดเท่าที่เคยมีมา lol

YEA

**Reply by JustinSolstice** [reply]
> I mean Psych Engine its just a good engine and you can choose between a bunch of engines which practically does the same stuff but every engine adds a lot of new functions

Thank you geez I'm tired of everyone saying like "Psych Engine is better" or "Kade engine is better", its just preference man just let the two engines live in peace

**Reply by ZHyproX** [reply]
the fact he stoled character editing from psych so lol

**Comment by Sebaxti0n** [comment]
there is a bug it keeps spamming the + buttton to turn up the volume i tried reseting my pc and it still does not work please fix this

**Reply by guizinhomartinez** [reply]
just go to the edit keybinds and add a keybind to the volume up and down

**Reply by OctoTheBitch** [reply]
Doesn't help

**Reply by kotsergey2** [reply]
Try changing the volume after that

**Reply by nenkim012** [reply]
> Doesn't help
dipunyaku berhasil aja

**Comment by just-pork** [comment]
Just hope it has HTML5 support

**Reply by Wire-Firey** [reply]
someone else ported 1.6.1 on Web Browser
https://github.com/Derpy-Jacob-903/KadeEngineWeb

**Comment by ID-SamSon** [comment]
Going into Freeplay just crashes the whole game for me.

**Reply by fcland** [reply]
re download or use 1.7 KE. BC. i dosnt have this problem



> Going into Freeplay just crashes the whole game for me.

re download or use 1.7 KE. BC. i dosnt have this problem

**Reply by RikkoDraws45** [reply]
1.7 keeps crashing when you are on the loading screen

**Reply by UngofaGamer** [reply]
> 1.7 keeps crashing when you are on the loading screen

The error off null object refrence?

**Comment by ghost** [comment]
The game appears to be spamming the ENTER button constantly, so I literally cannot get into a game without it pausing every frame. When setting keybinds, instead of the selected key (in this case: D,F,J,K), it showed up as "GUIDE."

**Reply by v0id96** [reply]
it happens to me too

**Reply by sosa360** [reply]
that sucks

**Comment by Serban-Smoke** [comment]
Idk what is the problem or how should I fix it but after install the source is good but when I add a song and go to freeplay is crash but when I add character and stage is work fine, and I don't know where I can change the health bar color.  When I try to reinstall the 1.7 back is crashing and I don't know why

**Comment by asdfd3v** [comment]
EPIC

**Comment by RikkoDraws45** [comment]
HI so when i open kade engine it seems to be spamming the volume button so that's basically all I hear...pls help

**Comment by Villezen** [comment]
Currently going to port my mod into this engine.

**Comment by StuffyAnt** [comment]
source/ModCore.hx:113: characters 48-71 : Type not found : polymod.FrameworkParams
source/Main.hx:20: lines 20-162 : Defined in this class
for more context here are the lines it's talking about

In ModCore.hx
![image](https://user-images.githubusercontent.com/88602380/140434876-542619be-47b5-4339-8d29-0d1613227cfa.png)
In Main.hx
![image](https://user-images.githubusercontent.com/88602380/140434896-0b850420-61ff-483b-b25b-1d5e5025a361.png)
am i missing something here?

**Comment by somefnfplayerlol** [comment]
this looks like version 2.0 because modcore

**Comment by Zicolie** [comment]
Bruh I can't build nor test it :(
![damnit](https://user-images.githubusercontent.com/86151568/141664172-f6514f7d-cda5-49d0-80f9-9852e4bbc3c0.png)
I'm kinda dumb so can someone tell me whats wrong, thanks :(

**Reply by itsbrxve** [reply]
You want to know what's wrong, EVERYTHING.

What did you do with the code exactly? You can use the output to bugfix your code.

**Reply by Wire-Firey** [reply]
he must looking the LUA Class files

**Comment by GamerXX1000GD** [comment]
It's pretty cool!

**Comment by inkthefurry** [comment]
it wont let me change the actual song it only plays the normal chart even after i have charted it i just started so i dont know what im doing

**Comment by amgous** [comment]
![hello chat](https://i.ibb.co/sgw68SY/Hello-chat.png)
what is this

**Comment by ghost** [comment]


**Reply by BenicioGPXD** [reply]
and who tf cares

**Comment by SusImpostor34** [comment]
YESSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

**Comment by MATIAS123456PRO** [comment]
siiiiiiii nueva acualizazion

**Comment by Wire-Firey** [comment]
Idea for 1.8.1 or 1.9: a Modchart editor on Chart Editor and Middle Scroll

**Reply by itsbrxve** [reply]
This engine is discontinued
